src/Symfony/Component/ClassLoader/README.md

```diff
@@ -10,28 +10,28 @@ standard or the PEAR naming convention.
 First, register the autoloader:
 
 ```php
-require_once __DIR__.'/src/Symfony/Component/ClassLoader/UniversalClassLoader.php';
+require_once __DIR__.'/src/Symfony/Component/ClassLoader/ClassLoader.php';
 
-use Symfony\Component\ClassLoader\UniversalClassLoader;
+use Symfony\Component\ClassLoader\ClassLoader;
 
-$loader = new UniversalClassLoader();
+$loader = new ClassLoader();
```

Comment:
I suggest submitting this readme change to older branches actually, to avoid showing the usage of deprecated classes in the readme.